code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -389,7 +389,7 @@
 /mob/living/silicon/ai/proc/ai_mob_to_structure()
 	disconnect_shell()
 	ShutOffDoomsdayDevice()
-	var/obj/structure/ai_core/deactivated/ai_core = new(get_turf(src))
+	var/obj/structure/ai_core/deactivated/ai_core = new(get_turf(src), /* skip_mmi_creation = */ TRUE)
```

Comment:
ditto

Reply:
Did this actually work?  Think it might be causing #68567

Reply:
it should do?